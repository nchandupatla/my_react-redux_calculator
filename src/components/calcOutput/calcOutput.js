import React from 'react';

import './calcOutput.css';

const calcOutput = (props) => (
        <div className="row">
        <div className="mdl-textfield mdl-js-textfield">
          <input id="calc-output" className="mdl-textfield__input" type="text" value={parseFloat(props.value)}  disabled />
        </div>
      </div>
);

export default calcOutput;