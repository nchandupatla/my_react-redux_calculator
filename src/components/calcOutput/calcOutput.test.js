import React from 'react';
import renderer from 'react-test-renderer';
import calcOutput from './calcOutput';


it('renders a snapshot', () => {
    const tree = renderer.create(<calcOutput/>).toJSON();
    expect(tree).toMatchSnapshot();
  });

  