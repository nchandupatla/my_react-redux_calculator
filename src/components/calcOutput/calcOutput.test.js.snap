exports[`test renders a snapshot 1`] = `
  <div className="row">
        <div className="mdl-textfield mdl-js-textfield">
          <input id="calc-output" className="mdl-textfield__input" type="text" value={parseFloat(props.value)}  disabled />
        </div>
      </div>
`;